consul_exporter.go

```diff
@@ -170,6 +177,7 @@ func NewExporter(opts consulOpts, kvPrefix, kvFilter string, healthSummary bool)
 		kvPrefix:      kvPrefix,
 		kvFilter:      regexp.MustCompile(kvFilter),
 		healthSummary: healthSummary,
+		logger:        log.NewLogfmtLogger(os.Stdout),
```

Comment:
You should modify the `NewExporter` function to add a `log.Logger` parameter and pass it here.